### PR TITLE
added new startup info

### PR DESCRIPTION
**Pull request by coder-oj**
Added info for another startup of kerlala 
hacktober fest 2022

### CODE REVIEW

**Comment by lemokami** [comment]
Hey @coder-oj, Thank you for the contribution✨. There is a small thing to point out. You haven't added the logo for entri.app and if you know please add the technologies(optional)

**Comment by AlexyPulivelil** [comment]
As mentioned I have made the changes.

File: data/entri.mdx [hunk 1]
```diff
@@ -0,0 +1,14 @@
+---
+name: entri.app
+description: Entri is a vernacular language learning platform for India, focused on helping people acquire relevant job skills. Entri is part of LearnLaunch, a leading EdTech accelerator based out of Boston, Massachusetts.
+founded: 2015
+based: Service Based
+domain: Edtech
+logo: entri_logo.png
```
Comment (lemokami) [inline data/entri.mdx:7]: @coder-oj you made a small mistake. The file name should be `entri_logo.jpg` but here it is `entri_logo.png`

**Review by lemokami — CHANGES_REQUESTED** [review]

**Comment by lemokami** [comment]
I have made a request for some changes. Hope it is understandable🙂. Don't forget to ⭐️ the repo

**Pull request closed without merging**